sdk/identity/azure-identity/src/environment_credential.cpp

```diff
@@ -44,20 +71,31 @@ EnvironmentCredential::EnvironmentCredential(TokenCredentialOptions options)
       }
       else
       {
+        PrintCredentialCreationLogMessage(
+            {
+                {AzureTenantIdEnvVarName, "tenantId"},
+                {AzureClientIdEnvVarName, "clientId"},
+                {AzureClientSecretEnvVarName, "clientSecret"},
```

Comment:
Is it safe to log clientSecret? Isn't that considered PII?

Reply:
We're not logging it, we logging the name, see https://github.com/Azure/azure-sdk-for-cpp/blob/51df426b557f8b29c2037aba53500418c73e8d07/sdk/identity/azure-identity/test/ut/default_azure_credential_test.cpp#L67-L70